packages/paste-website/stories/GenericHeader.stories.tsx

```diff
@@ -1,26 +1,27 @@
 import * as React from 'react';
-import {ComponentHeader} from '../src/components/shortcodes/component-header';
+import {GenericHeader} from '../src/components/shortcodes/generic-header';
 
 export const BasicHeader = (): React.ReactNode => {
-  return <ComponentHeader name="Alert" categoryRoute="/components" />;
+  return <GenericHeader name="Alert" categoryRoute="/components" />;
 };
 
 export const FullHeader = (): React.ReactNode => {
+  //QUESTION: why does FullHeader have opengraph meta tag, but not BasicHeader?
```

Comment:
It is because only component pages have open graph preview image thing

Reply:
I realized that it needs a packageName as a condition to render. BasicHeader doesn't have a packageName given, so no opengraph!